### ISSUE TITLE
Combine custom stylesheets for efficiency

### ISSUE THREAD

**Issue by crisner** [Google-Front-End-Weekend-Projects/A-Tribute-To-Stephen-Hawking#17]
It is good practice to combine custom styles into one main sheet. Please let me know if anyone is/will be working on this. If not I can do this by the end of the day. @Soumya44 I would like your approval to proceed with this.

**Comment by Soumya44** [comment]
Yup Crisner you can do that. Actually, I have been thinking about the same.

**Comment by crisner** [comment]
Great! I'll be working on it.
